components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5291,6 +5292,17 @@ def __init__(self, config, prefix, webIDLFile):
         # Do codegen for all the enums.
         cgthings = [CGEnum(e) for e in enums]
 
+        # Do codegen for all the typdefs
+        for t in typedefs:
+            if t.innerType.isUnion():
+                cgthings.extend([CGGeneric("\npub type %s = %s;\n\n" % (t.identifier.name,
+                                                                    "UnionTypes::" + str(t.innerType)))])
+            else:
+                assert not typeNeedsRooting(t.innerType, config.getDescriptorProvider)
+                cgthings.extend([CGGeneric("\npub type %s = " % (t.identifier.name)),
+                                getRetvalDeclarationForType(t.innerType, config.getDescriptorProvider()),
+                                CGGeneric(";\n\n")])
```

Comment:
This line and the line above need an extra space in front to align with the `[` above since they're in the list